content/learn/beginner/outputs-1.md

```diff
@@ -0,0 +1,85 @@
++++
+title = "Outputs 1"
+weight = 210
++++
+
+In this chapter, we will go over two ways to output text from a custom command: using the *response*, and later on using
+*template actions*.
+
+## Response
+
+If you paste text into the response field of a custom command, the bot will respond with that text when the command is
+triggered.
+
+For instance, paste the following text into a new custom command:
+
+```go
+Hello World.
+```
+
+This will make the bot respond "Hello World." in a new message when the command is triggered, as shown below.
+
+<center>
+
+![Hello World example ran in Discord.](hello_world.png)
+
+</center>
+
+## Template Actions
+
+### Actions As Output
+
+Earlier, we just made the bot respond with some static text. However, sometimes (or rather, most of the time) we want to
+change that around depending on various factors. This is where *template actions* come in.
+
+Template actions are a way to dynamically change the output depending on various things, such as the user who triggered
+the command, the arguments passed to the command, or even the current time.
+
+The bot evaluates the template action and replaces it with the result of the evaluation. For instance, the following
+code will make the bot respond with the server name when the command is triggered:
+
+```go
+{{ .Server.Name }}
+```
+
+As you can see, it differs from the static response in that it uses double curly braces (`{{` and `}}`) to denote an
+action. Without those, the bot would just respond with the text as-is.
```

Comment:
"The double curly braces (`{{` and `}}`) are essential: without these, the bot would simply respond with the text verbatim."

It may be worth adding a separate example illustrating this common pitfall, e.g., a command with the response `Hello .User.Username`.